frontend/src/layout/header/UserBox.tsx

```diff
@@ -226,9 +226,12 @@ const LoggedIn: React.FC<LoggedInProps> = ({ user }) => {
             }),
             children: t("user.logout"),
             css: {
-                color: COLORS.danger0,
-                ":hover, :focus": {
+                "&&": {
                     color: COLORS.danger0,
+                    ":hover, :focus": {
+                        color: COLORS.danger0,
+                    },
```

Comment:
Can... this be removed? Seems like a leftover. The color is already danger0...

Reply:
right, this can definitely be removed :D